src/components/FormattedText/FormattedText.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { ReactNode } from "react";
-import { Color, fontWeights, fontSizes } from "variables";
+import { Color } from "variables";
 import "styled-components/macro";
 
 type Props = {
```

Comment:
You have to export the props here to be able to import them above.
```suggestion
export type Props = {
```